### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).

It also includes the following components:

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by ann0see — APPROVED** [review]
I did the German part. Other languages need to be checked.

File: _translator-files/po/de/Installation-for-Linux.po [hunk 1]
```diff
@@ -80,53 +79,49 @@ msgstr "Debian und Ubuntu"
...
 #, no-wrap
 msgid "Using the official Repository (recommended)"
-msgstr ""
+msgstr "Nutze die offizielle Paketquelle (empfohlen)"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Linux.md:25
 msgid "We provide a repository for Debian and Ubuntu based distributions that allows automatic updates alongside your other system updates. To use the repository:"
-msgstr ""
+msgstr "Wir stellen Paketquellen für Debian und Ubuntu basierte Distributionen zur Verfügung, die automatische Upgrades neben den Systemupdates ermöglichen. Um diese zu verwenden folge diesen Schritten:"
```
Comment (ann0see) [inline _translator-files/po/de/Installation-for-Linux.po]: Sounds a bit clumsy

Comment (ann0see) [reply _translator-files/po/de/Installation-for-Linux.po]: Fixed.

**Comment by pljones** [comment]
https://hosted.weblate.org/translate/jamulus/1-index/en/?q=has%3Acheck&offset=3

Weblate thinks that needs translating.  Shouldn't it "know" better?  (I'm dubious about [the other two](https://hosted.weblate.org/zen/jamulus/1-index/en/?q=has:check) as well.)

**Comment by ann0see** [comment]
I think there's a way to silence the warning.

**Review by ignotus666 — APPROVED** [review]

**Comment by ann0see** [comment]
I'll merge this now. If anything is wrong, please correct it on weblate.

**Pull request merged**